packages/imperative/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ All notable changes to the Imperative package will be documented in this file.
 ## Recent Changes
 
 - Enhancement: Added a favicon to the Web Help that displays in browser tabs. [#801] (https://github.com/zowe/zowe-cli/issues/801)
+- Enhancement: Allowed the ProfileCredentials to check both (user and non-user) layers for secure fields. [#2460] (https://github.com/zowe/zowe-cli/issues/2460)
```

Comment:
- We should explain what ProfileCredentials is: "Allowed the ProfileCredentials method/highway/profile/whatever to check ..."
- I'd get rid of the parentheses: " ... to check the user and non-user layers ... "

Reply:
resolved in 6b3a273